pkg/kubelet/apis/config/helpers_test.go

```diff
@@ -156,6 +156,8 @@ var (
 		"CgroupDriver",
 		"CgroupRoot",
 		"CgroupsPerQOS",
+		"CloudConfigFile",
```

Comment:
this is not correct... this *is* a filepath

Reply:
true, moving to kubeletConfigurationPathFieldPaths and copying on kubeletConfigurationPathFieldPaths